wolfcrypt/src/chacha.c

```diff
@@ -112,25 +109,17 @@ int wc_Chacha_SetIV(ChaCha* ctx, const byte* inIv, word32 counter)
 {
     word32 temp[CHACHA_IV_WORDS];/* used for alignment of memory */
 
-#ifdef CHACHA_AEAD_TEST
```

Comment:
Did you mean to remove this code? I'm okay with it.

Reply:
@dgarske, yes, removal of debug code was intentional.